enterprise/backend/src/metabase_enterprise/sso/integrations/jwt.clj

```diff
@@ -140,7 +141,7 @@
   [{{:keys [jwt redirect]} :params, :as request}]
   (premium-features/assert-has-feature :sso-jwt (tru "JWT-based authentication"))
   (let [jwt-data (when jwt (session-data jwt request))
-        is-sdk? (sso-utils/is-embedding-sdk-header? request)]
+        is-sdk? (embed.util/has-react-sdk-header? request)]
```

Comment:
On master, I use `embed.util/is-modular-embedding-request?`. But since we don't have embedded analytics JS on 55, I only have a single function `embed.util/has-embedding-sdk-header?` in the `embedding.util` now.